container/container_contract.go

```diff
@@ -700,29 +729,30 @@ func Version() int {
 }
 
 func addContainer(ctx storage.Context, id, owner []byte, container Container) {
-	containerListKey := append(owner, id...)
+	containerListKey := append([]byte{ownerKeyPrefix}, owner...)
+	containerListKey = append(containerListKey, id...)
 	storage.Put(ctx, containerListKey, id)
 
-	common.SetSerialized(ctx, id, container)
+	idKey := append([]byte{containerKeyPrefix}, id...)
+	common.SetSerialized(ctx, idKey, container)
 }
 
 func removeContainer(ctx storage.Context, id []byte, owner []byte) {
-	containerListKey := append(owner, id...)
+	containerListKey := append([]byte{ownerKeyPrefix}, owner...)
+	containerListKey = append(containerListKey, id...)
 	storage.Delete(ctx, containerListKey)
 
-	storage.Delete(ctx, id)
+	storage.Delete(ctx, append([]byte{containerKeyPrefix}, id...))
 }
 
 func getAllContainers(ctx storage.Context) [][]byte {
-	var list [][]byte
+	list := [][]byte{}
```

Comment:
what's the byte code diff with that lines?

Reply:
Should be none, which means that the change is not needed.

Reply:
I think empty `var list [][]byte` returns `[{"type":"Any"}]` on stack, while `list := [][]byte{}` returns `[{"type":"Array","value":[]}]`.